helm-charts/helm2/strimzi-kafka-operator/templates/060-Deployment-strimzi-cluster-operator.yaml

```
@@ -36,6 +40,9 @@ spec:
           {{- end }}
           args:
             - /opt/strimzi/bin/cluster_operator_run.sh
+          volumeMounts:
+            - name: {{ .Values.logVolume }}
+              mountPath: /opt/strimzi/custom-config/
```

```suggestion
              mountPath: /opt/strimzi/log-config/
```

Custom config is a strange name, I'd probably call it log-config as we are always applying it (so not necessarily custom) and it only is relevant for logs.

---

`custom-config` is used in other operators, so I just wanted to keep it consistent.